cocos/platform/CCImage.h

```diff
@@ -144,7 +144,13 @@ class CC_DLL Image : public Ref
      @param    isToRGB        whether the image is saved as RGB format.
      */
     bool saveToFile(const std::string &filename, bool isToRGB = true);
-    
+
+    /**
+     * Enables or disables premultiplied alpha for PNG files.
+     *
+     *  @param enabled (default: true)
+     */
+    static void setPNGPremultipliedAlphaEnabled(bool enabled) { PNG_PREMULTIPLIED_ALPHA_ENABLED = enabled; }
```

Comment:
Please add static function before member functions as the rules in [coding styles](https://github.com/cocos2d/cocos2d-x/blob/v3/docs/CODING_STYLE.md#declaration-order).